tequilapi/endpoints/service.go

```diff
@@ -293,10 +305,14 @@ func (se *ServiceEndpoint) toServiceOptions(serviceType string, value *json.RawM
 }
 
 func toServiceInfoResponse(id service.ID, instance *service.Instance) serviceInfo {
+	proposal := instance.Proposal()
 	return serviceInfo{
-		ID:       string(id),
-		Status:   string(instance.State()),
-		Proposal: proposalToRes(instance.Proposal()),
+		ID:         string(id),
+		ProviderID: proposal.ProviderID,
```

Comment:
Now we have `ProviderID` and `ServiceType` twice in the same DTO. 
One day someone will change it in one place and will forget to change in another place. 
That is why I'd prefer to keep it only in one place.

Reply:
`proposal` is data structure of marketplace, and I expose fields, which caller used to start the service:
- I don't like that by exposing thru `proposal.ServiceType` we depends on inside effects what it matches now
- Is non-user-friendly for caller to find service by service type